staging/src/k8s.io/apiserver/pkg/authentication/config/v1alpha1/defaults.go

```diff
@@ -23,3 +23,9 @@ import (
 func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
+
+func SetDefaults_JWTAuthenticator(obj *JWTAuthenticator) {
+	if obj.ClaimMappings.Username.Claim == "" {
+		obj.ClaimMappings.Username.Claim = "sub"
```

Comment:
I think this needs a TODO to update it to check for the expression being set once we have the field.